docker/run-build

```diff
@@ -33,6 +33,14 @@ docker run --rm \
     --mount type=bind,source=${SOURCES_DIR},destination=/tmp/sources \
     --mount type=bind,source=${SCRIPTS_DIR},destination=/tmp/scripts \
     --mount type=bind,source=${WORK_DIR},destination=/tmp/work \
+    --mount type=bind,source=${ROOT_DIR}/.ccache,destination=/tmp/ccache \
+    -e CCACHE_DIR=/tmp/ccache \
+    -e USE_CCACHE=1 \
+    -e CCACHE_COMPRESS=1 \
+    -e CCACHE_COMPRESSLEVEL=9 \
+    -e CCACHE_CPP2=1 \
+    -e CCACHE_MAXSIZE=2G \
```

Comment:
1. Not sure is 2GB is enough for our builds.
2. May be it is better to change persistent ccache options via command line (`ccache --max-size=8G`, for instance)